srcpkgs/apparmor/template

```diff
@@ -103,7 +103,8 @@ libapparmor-devel_package() {
 	depends="${sourcepkg}-${version}_${revision}"
 	pkg_install() {
 		vmove usr/include/
-		vmove "usr/lib/*.a"
+		vmove usr/lib/*.a
+		vmove usr/lib/*.so
```

Comment:
I think `vmove` with wildcards should be quoted?

Reply:
I thought I read somewhere that it was the other way around but it makes sense and my memory most likely is wrong. Changed it.